### DISCUSSION TITLE
Screenshots of current Bisq2 state

### DISCUSSION THREAD

**Discussion by chimp1984** [bisq-network/bisq2#470] [General]
<img width="1512" alt="Screenshot 2022-08-26 at 21 52 11" src="https://user-images.githubusercontent.com/54558767/186982334-9a04ce24-ce6e-4c2e-8e84-fcafc13f4821.png">

<img width="1512" alt="Screenshot 2022-08-26 at 21 52 54" src="https://user-images.githubusercontent.com/54558767/186982357-a1fd0969-2dc1-456e-a054-2ff5cfdf0d71.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 52 59" src="https://user-images.githubusercontent.com/54558767/186982360-5ca72788-ea5a-4b58-88cf-b49525348615.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 53 04" src="https://user-images.githubusercontent.com/54558767/186982361-063e7808-26d7-409e-b7ff-5d6e2efe79b2.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 08" src="https://user-images.githubusercontent.com/54558767/186982363-2c600e36-e70b-418a-8365-d4b2523e5d86.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 11" src="https://user-images.githubusercontent.com/54558767/186982366-a7c85197-9f6b-494f-b6df-331ec298b4d0.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 19" src="https://user-images.githubusercontent.com/54558767/186982367-1821dd51-ecf5-4ba0-a294-5bb1d2f95227.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 32" src="https://user-images.githubusercontent.com/54558767/186982369-22162690-5012-429f-ab9a-16c74f66ebb0.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 40" src="https://user-images.githubusercontent.com/54558767/186982373-153cf41b-71d8-4645-8f53-636d7859d3b2.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 51" src="https://user-images.githubusercontent.com/54558767/186982375-ed6dcea2-941e-42fa-832d-ab9496316944.png">
<img width="1468" alt="Screenshot 2022-08-26 at 21 53 54" src="https://user-images.githubusercontent.com/54558767/186982376-ea5909c3-c1ce-44d7-b3f0-b878f1f9d0ba.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 54 03" src="https://user-images.githubusercontent.com/54558767/186982379-7c6cd187-c1bc-4974-b80f-0e49ab3fe8e1.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 54 06" src="https://user-images.githubusercontent.com/54558767/186982384-1ddec8a3-9620-4f5a-a3dd-5dd285f4247e.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 54 08" src="https://user-images.githubusercontent.com/54558767/186982389-da68fe43-8e89-43bd-be34-c714ad0e1daf.png">

<img width="1468" alt="Screenshot 2022-08-26 at 21 54 43" src="https://user-images.githubusercontent.com/54558767/186982395-cda94082-e250-4d42-9a6b-672f3b63aa2d.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 54 50" src="https://user-images.githubusercontent.com/54558767/186982396-c18a593f-2fd7-46a1-881f-8afea58d64d5.png">
<img width="1512" alt="Screenshot 2022-08-26 at 21 54 53" src="https://user-images.githubusercontent.com/54558767/186982398-59a38a0d-574c-46bc-8b9b-124e2c982fee.png">
<img width="1664" alt="Screenshot 2022-08-26 at 21 55 16" src="https://user-images.githubusercontent.com/54558767/186982401-cb3ad93c-36e5-4d88-a69e-14796f721c27.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 55 39" src="https://user-images.githubusercontent.com/54558767/186982403-6ac8c7ac-4381-4796-b00b-c81a88f95749.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 55 44" src="https://user-images.githubusercontent.com/54558767/186982404-707f2dd8-fe5a-4d2c-97f2-907b9d54be9d.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 55 51" src="https://user-images.githubusercontent.com/54558767/186982406-31cf413e-30d6-44b2-a454-c8541dc5398e.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 01" src="https://user-images.githubusercontent.com/54558767/186982410-fb8d9aed-0ad6-4812-b2e4-13f2170bcac5.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 08" src="https://user-images.githubusercontent.com/54558767/186982412-145e5dd5-6e51-455d-bbcf-4bfbdbd42fc4.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 11" src="https://user-images.githubusercontent.com/54558767/186982415-dfcd4665-d5b8-4497-b978-8e0d88507614.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 17" src="https://user-images.githubusercontent.com/54558767/186982418-3f5dfcca-d5af-4a46-a53b-42b0a4f0f94f.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 24" src="https://user-images.githubusercontent.com/54558767/186982419-add0ea87-de96-425b-ac14-c2deac47126b.png">
<img width="1431" alt="Screenshot 2022-08-26 at 21 56 27" src="https://user-images.githubusercontent.com/54558767/186982421-99825a5c-37ac-426d-8c3b-fec85bc2b0ef.png">
<img width="1431" alt="Screenshot 2022-08-26 at 21 56 32" src="https://user-images.githubusercontent.com/54558767/186982422-6009bd5e-d8ca-4ae1-ad3d-32d2f30dd2b5.png">
<img width="1431" alt="Screenshot 2022-08-26 at 21 56 35" src="https://user-images.githubusercontent.com/54558767/186982424-adee8e3c-dfd3-40ad-bab8-5d66c38b2fd8.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 45" src="https://user-images.githubusercontent.com/54558767/186982425-e9e992c8-e146-4d33-803f-4ac753f61349.png">
<img width="1475" alt="Screenshot 2022-08-26 at 21 56 48" src="https://user-images.githubusercontent.com/54558767/186982426-2bdeb2a1-bbe3-4f47-974f-4bae5e5a68e9.png">

**Comment by Conza88** [comment]
- Public profile: what does this effect? can I change it later? is it a anon thing? What is the recommendation from Bisq? e.g. I should use a new pseudonym, or one with prior reputation may help? Can this be changed later?
All come to mind at that screen.
- Welcome to bisq - "more on the way" - 'monero swaps' (* great, dev work on not bitcoin stuff - 🤮). Mild /s, but if'd be ideal if there was a way maybe prior to this; tailoring to who the user is? bitcoin only, 'crypto trader', privacy whatever... 'how would you describe yourself'? or something? 
Bitcoin only.. then there's a default setup that has all shitcoins removed... so I don't need to do that manually later.
- Enter bisq easy - like this. Is the 'exchange trade details in a private chat with your peer' - there a template that is requested/or provided? or nothing? Also like the 'learn & discover' section.
- trade apps - can unpin, remove eg. 'monero swaps' , right? tailor that?
- "select a market" = you can select more than one, right? 🤔
- reputation - interesting. more info on this? 
- Any notifications - bisq 2 can send; or at least app - traders can easily chat about? 
- Priorities - bisq multisig should be before liquid swaps and monero imo.
- liquid swaps = good write up at least
- If burning bsq - is best for reputation; surely bsq atomic swaps should be higher in priority than liquid etc.
- burning bsq - have to go to bisq 1 to do?

**Comment by flix1** [comment]
This is looking amazing!

![image](https://user-images.githubusercontent.com/4233784/193664844-a1ede644-e64a-4244-98f6-2edc5625fa9c.png)

I can't wait for a release date.

1. Bisq Easy
2. Liquid Swaps
3. Monero Swaps
4. Bisq Multisig
5. Lightning X
6. BSQ swaps

As well as these trade protocols I'd love to see experiments with other security systems to see how they compare.

Bisq 2of3 multisig with arbitration (old old Bisq)
Bisq MAD
Bisq 2of2 multisig with timelocks (less extreme than MAD)
Bisq 2of2 multisig with optional arbitration post failed trade (both users can agree to go to arbitration and sign a tx sends funds to 2of3)
Bisq 2of2 with penalty for non-responding party
Bisq where BSQ bonds are required to trade instead of BTC security deposits. If trade fails BSQ bond is burnt as penalty.

etc, etc...

In fact there should some way to experiment with all of these in an adversarial environment.
